packages/open-next/src/build/installDeps.ts

```diff
@@ -0,0 +1,50 @@
+import { execSync } from "child_process";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+import { InstallOptions } from "types/open-next";
+
+import logger from "../logger.js";
+
+export function installDependencies(
+  outputDir: string,
+  installOptions?: InstallOptions,
+) {
+  try {
+    if (!installOptions) {
+      return;
+    }
+    const name = outputDir.split("/").pop();
+    // First we create a tempDir
+    const tempInstallDir = fs.mkdtempSync(
+      path.join(os.tmpdir(), `open-next-install-${name}`),
+    );
+    logger.info(`Installing dependencies for ${name}...`);
+    // We then need to run install in the tempDir
+    // We don't install in the output dir directly because it could contain a package.json, and npm would then try to reinstall not complete deps from tracing the files
+    const installCommand = `npm install --arch=${installOptions.arch ?? "arm64"} --platform=linux --target=${installOptions.nodeVersion ?? "18"} --libc=${installOptions.libc ?? "glibc"} ${installOptions.packages.join(" ")}`;
```

Comment:
Do we want to use whatever packager was used for Next as it has been retrieved anyway?

Reply:
Maybe, but we'll need to figure out how to pass all these options. 
I don't think they support all of them, or there is no docs. If some other package manager support it we could use them and fallback to npm when not possible

Reply:
We could pass the `buildOptions` and the function name to `installDeps`.
It would mean adding the `outputPath` and the packager there.
But that's probably a nice to have for later.

Reply:
I thought about using `buildOptions` and `name`, but i think it would just overcomplicate things.
It could be a splitted functions (which are under functions in the config) and server functions are inside a different folder than the rest.
We could still pass `buildOptions` to get the packager